foundations/javascript_basics/fundamentals-1.md

```diff
@@ -106,13 +106,15 @@ This section contains questions for you to check your understanding of this less
 * <a class="knowledge-check-link" href="#variable-declaration">Name the three ways to declare a variable</a>
 * <a class="knowledge-check-link" href="#avoid-var">Which of the three variable declarations should you avoid and why?</a>
 * <a class="knowledge-check-link" href="https://javascript.info/variables#variable-naming">What rules should you follow when naming variables?</a>
-* <a class="knowledge-check-link" href="https://javascript.info/operators#string-concatenation-with-binary">What should you look out for when using the `+` operator with numbers and strings?</a>
-* <a class="knowledge-check-link" href="https://javascript.info/operators#remainder">How does the `%` operator work?</a>
-* <a class="knowledge-check-link" href="https://www.w3schools.com/js/js_numbers.asp">Explain the difference between `==` and `===`.</a>
-* <a class="knowledge-check-link" href="https://www.w3schools.com/js/js_numbers.asp">When would you receive a `NaN` result?</a>
-* <a class="knowledge-check-link" href="https://javascript.info/operators#increment-decrement">How do you increment and decrement a number?</a>
-* <a class="knowledge-check-link" href="https://javascript.info/operators#increment-decrement">Explain the difference between prefixing and post-fixing increment/decrement operators.</a>
+* <a class="knowledge-check-link" href="https://javascript.info/operators#string-concatenation-with-binary">What happens when you add numbers and strings together?</a>
+* <a class="knowledge-check-link" href="https://javascript.info/operators#remainder">How does the Modulus(%)* operator work?</a>
```

Comment:
Can we change to Modulo?

Also, instead of putting an asterisk, just put the "remainder" hint inline.

`How does the Modulo (%), or Remainder, operator work?`

Reply:
Sure, I've changed the sentence and added the hint inline.